examples/server/server.cpp

```diff
@@ -217,6 +221,32 @@ struct whisper_print_user_data {
     int progress_prev;
 };
 
+bool convert_to_WAV(const std::string& temp_filename, std::string &error_resp) {
+    std::ostringstream cmd_stream;
+    std::string converted_filename_temp = temp_filename + "_temp.wav";
+    cmd_stream << "ffmpeg -i \"" << temp_filename << "\" -ar 16000 -ac 1 -c:a pcm_s16le \"" << converted_filename_temp << "\" 2>&1";
+    std::string cmd = cmd_stream.str();
+
+    int status = std::system(cmd.c_str());
+    if (status != 0) {
+        error_resp = "{\"error\":\"FFmpeg conversion failed. Check if FFmpeg is installed\"}";
```

Comment:
I got an idea, we can put this check before we even start the server. And let the user who starts the server up know that there is no ffmpeg and perhaps exit. Then we don't need to check for it for every request.

Reply:
Yup!